relayer/pkg/starknet/feeder_client.go

```diff
@@ -0,0 +1,211 @@
+package starknet
```

Comment:
most code was repurposed from juno (they didn't support some of the response fields)